content/docs/music.md

```diff
@@ -73,5 +96,14 @@ The noise channels gets the most instruments, but that's also because of how the
 - You should set the instrument upon a volume change on the wave channel.
 - To prevent notes that overlap when looping your song in-game, enter `00` as the note's volume on the tick it should stop playing.
 - If you're not using the pulse channels right as the song starts, it is a wise idea to mute them using a C00 effect. They sometimes start out by playing garbled notes, even if there's nothing present on those channels.
+- You may notice that the channels, when previewing your song within your tracker, will be already panned left and right. This is a limitation of the .MOD file format itself and it isn't something you can change. The default panning will not be present upon conversion, unless set by yourself via effects (more info below).
```

Comment:
I think we have too many tips for one list. It doesn't need to change for now, but I'm thinking this stuff could be condensed into easy-to-read bullet points on a per-software basis (OpenMDT Tips, MilkyTracker Tips, General Tips) and also to keep the really important stuff (like channel 3 needing its instrument on volume) up by Important Limitations so people already familiar with trackers know what to do.